quarkchain/cluster/tests/test_grpc_client.py

```diff
@@ -0,0 +1,59 @@
+import grpc
+import unittest
+
+from concurrent import futures
+from quarkchain.generated import grpc_pb2
+from quarkchain.generated import grpc_pb2_grpc
+from quarkchain.cluster.grpc_client import GrpcClient
+
+
+class NormalServer(grpc_pb2_grpc.ClusterSlaveServicer):
+    def SetRootChainConfirmedBlock(self, request, context):
+        return grpc_pb2.SetRootChainConfirmedBlockResponse(
+            status=grpc_pb2.ClusterSlaveStatus(code=0, message="Confirmed")
+        )
+
+
+class ErrorServer(grpc_pb2_grpc.ClusterSlaveServicer):
+    def SetRootChainConfirmedBlock(self, request, context):
+        return grpc_pb2.SetRootChainConfirmedBlockResponse(
+            status=grpc_pb2.ClusterSlaveStatus(code=1, message="Confirmed")
+        )
+
+
+class TestGrpcClient(unittest.TestCase):
+    def build_test_server(self, test_server, port: int):
+        server = grpc.server(futures.ThreadPoolExecutor(max_workers=10))
+        grpc_pb2_grpc.add_ClusterSlaveServicer_to_server(test_server(), server)
+        server.add_insecure_port("[::]:" + str(port))
+        return server
+
+    def test_exception_error(self):
+        server_port = 50051
+        server = self.build_test_server(NormalServer, server_port)
+        server.start()
+
+        client_host = "localhost"
+        client_port = 50011
+        resp = GrpcClient(client_host, client_port).set_rootchain_confirmed_block()
+        self.assertFalse(resp)
```

Comment:
seems this test doesn't need a server at all, right?

Reply:
also add a comment explaining we expect an connection exception for this test, and that's why the result is `False`